awx/settings/defaults.py

```diff
@@ -476,7 +476,7 @@
 
 # Django Caching Configuration
 DJANGO_REDIS_IGNORE_EXCEPTIONS = True
-CACHES = {'default': {'BACKEND': 'awx.main.cache.AWXRedisCache', 'LOCATION': 'unix:/var/run/redis/redis.sock?db=1'}}
+CACHES = {'default': {'BACKEND': 'awx.main.cache.AWXRedisCache', 'LOCATION': 'unix://var/run/redis/redis.sock?db=1'}}
```

Comment:
Just noting here so it doesn't get lost

```suggestion
CACHES = {'default': {'BACKEND': 'awx.main.cache.AWXRedisCache', 'LOCATION': 'unix:///var/run/redis/redis.sock?db=1'}}
```

Otherwise I think the path will be relative, it'll get parsed as `unix://` and `var/run/redis/redis.sock?db=1` so the path will be relative to the cwd.